CONTRIBUTING.md

```diff
@@ -194,11 +161,64 @@ you can change the base branch using `git rebase --onto`, like this:
   $ git rebase --onto <NEW-BASE-BRANCH> <OLD-BASE-BRANCH> <YOUR-BRANCH>
 ```
 
-For example, backporting `my_branch` from `master` to `3.x`:
+For example, backporting `my_branch` from `master` to `4.x`:
 
 ```bash
-  $ git rebase --onto 3.x master my_branch
+  $ git rebase --onto 4.x master my_branch
+```
+
+### Making contributions that depend on pull requests in spyder-kernels
```

Comment:
@impact27, I improved the instructions in this section using your suggestions. Please take a look at them and let me know what you think.

Reply:
You could start with a description of the system:

Spyder and spyder-kernel versions are linked because a lot of communication happens between them. For this reason, a clone of spyder-kernel is placed in the `external-deps` subfolder of the spyder repository. These instructions will help you if you need to make changes that touches both repository.